adapter_author_deps.bzl

```diff
@@ -0,0 +1,57 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_repository")
+load('@bazel_tools//tools/build_defs/repo:git.bzl', 'git_repository')
```

Comment:
No need to load `git_repository`, it can be called by `native.git_repository` instantly.